src/generator/com/elovirta/pdf/front-matter.xsl

```diff
@@ -39,10 +39,9 @@
 <!--            </axsl:choose>-->
         </fo:block>
         <!-- set the subtitle -->
-        <axsl:apply-templates select="$map//*[contains(@class,' bookmap/booktitlealt ')]"/>
-<!--          <fo:block axsl:use-attribute-sets="__frontmatter__owner">-->
-<!--            <axsl:apply-templates select="$map//*[contains(@class,' bookmap/bookmeta ')]"/>-->
-<!--          </fo:block>-->
+        <fo:block axsl:use-attribute-sets="__frontmatter__titlealt">
```

Comment:
In PDF2, the element `<booktitlealt>` uses attribute set `__frontmatter__subtitle`. Poor naming, but it hasn't been changed for backwards compatibility. So the implementation should use `style-cover-titlealt` but map to `__frontmatter__subtitle`. That wasy this wrapper element would not be needed. Also, if there is no `<booktitlealt>`, we'd still get the wrapper.